README.md

```diff
@@ -1,4 +1,58 @@
 Checkmate
 =====
 
-A service for checking URLs are safe.
\ No newline at end of file
+A service for checking URLs are safe.
+
+Installing Via HTML in a development environment
+------------------------------------------------
+
+### You will need
+
+* Via HTML integrates with h and the Hypothesis client, so you will need to
```

Comment:
```suggestion
* Checkmate integrates with h and the Hypothesis client, so you will need to
```

But also, is this bullet-point accurate? I don't think Checkmate is gonna depend on any of our other apps. The Via apps depend on Checkmate, but that's something for their READMEs to state. I don't think Checkmate itself has any dependencies on other apps